semantic-python/semantic-python.cabal

```diff
@@ -18,17 +18,33 @@ extra-source-files:  README.md
 
 tested-with:         GHC == 8.6.4
 
-library
-  exposed-modules:
-    Language.Python.Core
-  -- other-modules:
-  -- other-extensions:
-  build-depends:       base ^>=4.12.0.0
+common haskell
```

Comment:
As long as we’re making a common stanza, can we add warnings to it?

Reply:
Done.